src/components/aria-live/aria-live.spec.ts

```diff
@@ -0,0 +1,95 @@
+import {
```

Comment:
Can do that, but it seems like you're only compiling styles within `src/core/styles`, I can add the component to BrocolliSass as well, or we keep it temporary in the `src/components`